tests/by-util/test_tac.rs

```diff
@@ -205,3 +205,21 @@ fn test_null_separator() {
         .succeeds()
         .stdout_is("b\0a\0");
 }
+
+#[test]
+fn test_regex() {
+    new_ucmd!()
+        .args(&["-r", "-s", ":+"])
+        .pipe_in("a:b::c:::d::::")
+        .succeeds()
+        .stdout_is("d::::c:::b::a:");
+}
+
+#[test]
+fn test_regex_before() {
```

Comment:
would it be possible to add some more complex regex to testing?

Reply:
Sure, I can add some more.

Reply:
ping?